README.md

```diff
@@ -35,33 +41,80 @@ To see the graphics, you can open another terminal and run
 ```bash
 GAZEBO_MODEL_PATH=`pwd`/../python/social_bot/models gzclient
 ```
+#### [Grocery Ground Test](python/social_bot/envs/grocery_ground.py)
+```bash
+python3 python/social_bot/envs/grocery_ground.py
+```
 
 ## Environments and Tasks
 
 ### [Simple Navigation](python/social_bot/envs/simple_navigation.py)
 
 * [Goal task](python/social_bot/teacher_tasks.py). 
     
-    To be updated
+    Gif to be updated    
 
 ### [Grocery Ground](python/social_bot/envs/grocery_ground.py)
 
-* [Goal task](python/social_bot/teacher_tasks.py). 
+* [Goal task](python/social_bot/envs/grocery_ground.py). 
 
-    <img src="examples/grocery_ground_pioneer.gif" width="320" height="240" alt="pioneer"/>
+    <img src="media/grocery_ground_pioneer.gif" width="320" height="240" alt="pioneer"/>
+
+    A task to chase a goal. 
+    
+    The agent will receive reward 1 when it is close enough to the goal, and get -1 if moving away from the goal too much or timeout.
 
 ### [PR2 Gripping](python/social_bot/envs/pr2.py)
 
 * Gripping task
 
-    To be updated
+    Gif To be updated
+
+    A task that the agent need to use its grippers or fingers to grip a beer.
+
+    A simple reward shaping is used to guide the agent's gripper to get close to the target, open the gripper and lift the target up.
+
 
 ### [iCub Walking](python/social_bot/envs/icub_walk.py)
 
 * Humanoid walking task
 
-    <img src="examples/icub_walk.gif" width="320" height="240" alt="pioneer"/>
+    <img src="media/icub_walk.gif" width="320" height="240" alt="pioneer"/>
+
+    A task that the agent learns how to walk. 
+    
+    reward = not_fall_bonus + truncked_walk_velocity - ctrl_cost
+ 
 
+## Training Examples
+### [Simple Navigation Training Example](https://github.com/HorizonRobotics/alf/blob/master/alf/examples/ac_simple_navigation.gin)
+Training with [Agent Learning Framework (Alf)](https://github.com/HorizonRobotics/alf)
+```bash
+cd ALF_REPO_ROOT/examples/
+python -m alf.bin.main --root_dir=~/tmp/simple_navigation --gin_file=ac_simple_navigation.gin --alsologtostderr
+```
+### [PR2 Training Example](https://github.com/HorizonRobotics/alf/blob/master/alf/examples/ppo_pr2.gin)
+Training with Alf:
+```bash
+python -m alf.bin.main --root_dir=~/tmp/pr2_gripping --gin_file=ALF_REPO_ROOT/examples/ppo_pr2.gin --alsologtostderr
+```
+
+### [Grocery Ground Training Example](examples/grocery_alf_ppo.gin)
+Training with Alf:
+```bash
+python -m alf.bin.main --root_dir=~/tmp/grocery_ppo --gin_file=grocery_alf_ppo.gin --alsologtostderr
+```
+
+### [iCub Walking Task Training Example](examples/icub_alf_ppo.gin)
+Training with Alf:
+```bash
+python -m alf.bin.main --root_dir=~/tmp/icub_ppo --gin_file=icub_alf_ppo.gin --alsologtostderr
+```
+Or you can train it with tf-agent SAC, which might be better at sample efficiency:
```

Comment:
Can you also try with ALF SAC?

Reply:
Sure, I'll try

Reply:
Need some tests, maybe latter in another PR